.github/workflows/std-libs-benchmark.yml

```diff
@@ -18,12 +18,6 @@ jobs:
     runs-on:
       - benchmark
     steps:
-      - if: startsWith(runner.name, 'GitHub Actions') || startsWith(runner.name, 'Hosted Agent')
```

Comment:
Removing `conda` at so many places is great, but shouldn't there also be a removal of the usage of `code` somewhere?